src/PeliasMapkitExtensions.swift

```diff
@@ -144,3 +144,10 @@ extension SearchBoundaryRect {
     self.minLatLong = GeoPoint(latitude: maxCoordinate.latitude, longitude: minCoordinate.longitude)
   }
 }
+
+public extension GeoPoint {
```

Comment:
@ecgreb is correct, and Pelias is built to not require CoreLocation (same for Tangram-es) so we need a structure that allows for that.

One of the plans I have post-1.0 is to build a "Mapzen Utilities" library that all 3 of these would depend upon (cuz On the Road has its own version as well) and would utilize.